cmd/docs/main.go

```diff
@@ -42,7 +46,10 @@ hide:
 	cdd := viper.GetString(commandDocsDir)
 
 	// automatically generate markdown documentation for all Iter8 commands
```

Comment:
Automatic markdown generation can be removed from basecli and absorbed into k8s cli.